src/components/gataca-qrws/readme.md

```diff
@@ -0,0 +1,154 @@
+# Gataca-QR
+
+This component built using stencyl allows an easy integration to display a gataca QR.
+It allows to integrate 2 slots, named "title" and "description", to provide further integration to the user upon display of the QR.
```

Comment:
Custom component?

Reply:
The QR WS text is the old.
Anything WS is WIP as stated on the title of the PR.